src/components/colors/hex.js

```diff
@@ -0,0 +1,11 @@
+import colors from './colors';
+
+const hex = {};
+
+Object.keys(colors).forEach(groupName => {
```

Comment:
I would export is as a function

Reply:
iife?

Reply:
nah, more like colors module could export function `getHexColors()` etc

but it's random thought, it's okay this way too

Reply:
It's a bit misleading, as colors already have `hex` property